packages/components/src/external-link/styles/external-link-styles.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+
+/**
+ * WordPress dependencies
+ */
+import { Icon } from '@wordpress/icons';
+
+export const StyledIcon = styled( Icon )`
+	width: 1.4em;
+	height: 1.4em;
+	margin: -0.2em 0.1em 0;
```

Comment:
I'm pretty sure these need special care for RTL.

Reply:
Does this actually need it? The values as written here will have equal margin for left and right as the CSS values go "top, right, bottom, left". As "left" is omitted, the value for "right" will be used, making it equal on both sides. Is my understanding correct that we don't need to make a change for RTL in this case? If not then I will need some direction from someone more knowledgable than me to know what needs to be changed here.

Reply:
Seems like you're right here. no particular change is needed here.